Moonscraper Chart Editor/Assets/Scripts/Game/Charts/IO/Midi/MidIOHelper.cs

```diff
@@ -26,6 +26,7 @@ public static class MidIOHelper
         // Note numbers
         public const byte DOUBLE_KICK_NOTE = 95;
         public const byte SOLO_NOTE = 103;                 // http://docs.c3universe.com/rbndocs/index.php?title=Guitar_and_Bass_Authoring#Solo_Sections
+        public const byte TAP_NOTE = 104;                  // https://strikeline.myjetbrains.com/youtrack/issue/CH-390
```

Comment:
It is CH only currently, but I implemented it for reading only, not exporting. And it is implemented in CH, the issue is outdated regarding that.

Reply:
Okay is that case let's change this to
```suggestion
        public const byte CH_TAP_NOTE = 104;                  // Link to Nathan's doc
```
as the midi readers and writers were originally targeting RB style midi parsing, so this would make it a bit clearer that it's an exception to that spec. I also don't trust that the issue won't be closed or deleted once CH development comes to a close so your doc is a reasonable substitute.

Reply:
My only problem with renaming the constant is that none of the other non-RB stuff does this, would you want me to change that?

Reply:
Looks like your reply got misplaced lol, ended up in the comments

Alrighty, I've added a prefix to just this in that case. Not too big a fan of it, though.

Reply:
Yeah, I tried to use email replies on my phone hoping it would link back to this comment, that experiment's failed. :P